google/cloud/forseti/services/scanner/dao.py

```diff
@@ -123,19 +126,20 @@ def create(self, violations, inventory_index_id):
                         violation.get('violation_data', ''),
                     )
 
-                    violation = self.TBL_VIOLATIONS(
+                    violation = Violation(
+                        created_at_datetime=created_at_datetime,
+                        full_name=violation.get('full_name'),
                         inventory_index_id=inventory_index_id,
+                        resource_data=violation.get('resource_data'),
                         resource_id=violation.get('resource_id'),
                         resource_type=violation.get('resource_type'),
-                        full_name=violation.get('full_name'),
-                        rule_name=violation.get('rule_name'),
                         rule_index=violation.get('rule_index'),
-                        violation_type=violation.get('violation_type'),
+                        rule_name=violation.get('rule_name'),
+                        scanner_start_time=scanner_start_time,
```

Comment:
Scanner time should not be used here. 
What should be here is `scanner_index_id`, which references a new `scanner_index` table. 
This is very analogous to the `inventory_index` table and how it's tied to the `gcp_inventory` table, via the `inventory_index_id`.

Reply:
Gotcha. Will rework the branch accordingly.